superset-frontend/src/views/CRUD/data/hooks.ts

```diff
@@ -83,3 +83,10 @@ export const UseGetDatasetsList = (queryParams: string | undefined) =>
     .catch(error =>
       logging.error('There was an error fetching dataset', error),
     );
+
+export const UseGetDatasetRelatedObjects = (id: string) =>
```

Comment:
nit, but convention on any methods is to have them in camel case with the first letter lowercase. Can you fix the UseGetDatasetsList as well?

Reply:
I updated the dataset hooks as discussed in Slack in [`this commit`](https://github.com/apache/superset/pull/22043/commits/4c2efaff40ec02caf4c8ec918af06a0203b722d8).